pages/product.js

```diff
@@ -0,0 +1,20 @@
+import { Component } from "react";
+
+export class product extends Component {
+  constructor() {
+    super();
+    this.state = {
+      item: "This is the product",
```

Comment:
Don't just add state to a component. Whenever you add state, you should have a very good reason for doing so. Think about application state. What is it? What is the application state for your app?

This is something that is not easy to learn. You just need practice, but this is not state.